frontend/src/components/Banner/Banner.tsx

```diff
@@ -50,7 +51,10 @@ export const Banner = ({
       <Box __css={styles.banner}>
         <Flex sx={styles.item}>
           <Flex>
-            <Icon as={BxsInfoCircle} __css={styles.icon} />
+            <Icon
+              as={variant === 'info' ? BxsInfoCircle : BxsErrorCircle}
+              __css={styles.icon}
+            />
             <ReactMarkdown components={mdComponents}>{children}</ReactMarkdown>
```

Comment:
As stated, should change this component to accept any children, or only markdown with `useMarkdown`. Do you want to do it or should I do it in a separate PR?

Reply:
edited, same as above